### ISSUE TITLE
Unable to return null from "evaluate"

### ISSUE THREAD

**Issue by agrinevsky** [tableau/TabPy#380]
If my script returns null (None) as the result, I'd expect to receive json null in the response, but get an error instead.

Example request:
```
HTTP POST localhost:9004/evaluate
{
   "data" : {},
   "script": "return None"
}
```

Response:
```
{
    "message": "Error running script. No return value",
    "info": {}
}
```

**Comment by phantomcosmonaut** [comment]
ExceptionPlaneHandler line 94:
`if result is None:
        self.error_out(400, 'Error running script. No return value')
else:
        self.write(json.dumps(result))
        self.finish()`

This error_out seems intentional, but if there is no valid reason behind keeping it, then you could just remove the if/else statement and json.dumps will return "null"

**Comment by 0golovatyi** [comment]
@phantomcosmonaut Yes, this is pretty much what needs to be done to fix the issue. Plus unit test for the case :)

**Comment by phantomcosmonaut** [comment]
I’d be happy to help out, should I commit to master branch?

**Comment by 0golovatyi** [comment]
@phantomcosmonaut Yes, PR for `origin:master` is how we recommend changed to be added to the repo.

**Comment by 0golovatyi** [comment]
The fix is added in https://github.com/tableau/TabPy/pull/388.

**Comment by 0golovatyi** [comment]
The fix is in published package v0.9.0.
